code/modules/detectivework/detective_work.dm

```diff
@@ -9,7 +9,7 @@
 			if(add_blood_from_data(G.bloody_hands_data)) //only reduces the bloodiness of our gloves if the item wasn't already bloody
 				G.transfer_blood--
 				M.update_inv_gloves()
-	else if(M.bloody_hands)
+	else if(M.bloody_hands > 1)
```

Comment:
surely >=?

Reply:
It's > 1 so that it won't spread the last bit of blood from the hand to the thing being touched.
Otherwise it removes all of the blood from the hands.